CodeStarterCamp_Week2/main.swift

```diff
@@ -10,3 +10,21 @@ import Foundation
 
 print("Hello, World!")
 
+var chooseNumberSet = Set<Int>()
```

Comment:
변수 네이밍은 보통 명사로 시작합니다.
동사가 들어가야하는 경우에는 동사 choose를 과거분사로 만들어서 명사 number을 수식하는 형태로 만들면 좋습니다. 따라서 chosenNumberSet로 할 수 있겠네요.

그리고 변수나 함수 네이밍시 타입에대한 정보는 네이밍에 포함시키지 않는 것이 좋습니다.
관련해서 [Swift ApI Guidelines](https://www.swift.org/documentation/api-design-guidelines/) 참고 부탁드립니다.
Naming - Promote Clear Usage - Name variables, parameters, and associated types according to their roles, rather than their type constraints. 부분
-> chosenNumberSet에서 Set이 타입 이름이니까 chosenNumbers 정도로 바꿀 수 있을 것 같습니다.